### ISSUE TITLE
Option to gate progression

### ISSUE THREAD

**Issue by Leadenigma** [StrangeLoopGames/EcoSuggestions#1642]
Would love to see an option on server settings to gate progression based on certain factors or an age system. stone age, steam age, Industrial age, Modern age.

e.g., Locking Tier 2 professions until 50% - 60% of the active population have researched or learnt tier 1 skills and so on, or time based , after a certain amount of days certain professions open up.

**Comment by jlai** [comment]
You can probably do this through a set of laws (pre-populated before anyone joins the server). Or through modding.

**Comment by MonzUn** [comment]
Yeah, both of those routes are possible and are already applied across the community. So I would rather devs spend time on higher priority tasks that we cna't solve as easily via civics or mods :)
